components/webdriver_server/lib.rs

```diff
@@ -146,9 +168,12 @@ pub struct WebDriverSession {
     unhandled_prompt_behavior: String,
 
     // https://w3c.github.io/webdriver/#dfn-input-state-table
-    input_state_table: HashMap<String, InputSourceState>,
+    input_state_table: RefCell<HashMap<String, InputSourceState>>,
+
     // https://w3c.github.io/webdriver/#dfn-input-cancel-list
```

Comment:
As you are modifying these lines, I think it might be nice to fix the rustdoc for these members as well:

```rust
/// <https://w3c.github.io/webdriver/#dfn-input-state-table>
```